Exercises/NeedForSpeed/RaceTrack.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exercises.NeedForSpeed
+{
+    class RaceTrack
+    {
+        private int _distance;
+
+        public RaceTrack(int distance)
+        {
+            _distance = distance;
+        }
+
+        public bool TryFinishTrack(RemoteControlCar controlCar)
+        {
+            while (!controlCar.BatteryDrained())
+            {
+                controlCar.Drive();
+            }
+            if (controlCar.DistanceDriven() >= _distance)
```

Comment:
Pensa comigo: o que o código `controlCar.DistanceDriven() >= _distance` retorna? Um booleano, correto? Nesse caso, você poderia retornar ele direto, sem o if (já que, caso a condição for verdadeira, ele retornará um `true`, e caso seja falsa, retornará um `false`, exatamente como o seu if está fazendo).